nuts/plugin.py

```diff
@@ -19,7 +19,7 @@
 def nuts_ctx(request: FixtureRequest) -> NutsContext:
     params = request.node.params
     context_class = getattr(request.module, "CONTEXT", NutsContext)
-    ctx = context_class(params, pytestconfig=request.config)
+    ctx = context_class(params, pytestconfig=request.config)  # type: NutsContext
```

Comment:
I think you might have written this better as: `ctx: NutsContext = context_class(params, pytestconfig=request.config)`?
that's [how variable declarations are annotated](https://mypy.readthedocs.io/en/stable/type_inference_and_annotations.html).

Reply:
Oh, thanks for the hint :-)